velox/serializers/PrestoSerializer.cpp

```diff
@@ -752,6 +759,11 @@ void readArrayVector(
     velox::memory::MemoryPool* pool,
     const SerdeOpts& opts,
     VectorPtr& result) {
+  if (result->encoding() != VectorEncoding::Simple::ARRAY) {
+    SelectivityVector rows(resultOffset, false);
```

Comment:
any reason we use this here instead of `SelectivityVector::empty()` everywhere else?

Reply:
Good catch, this was an earlier attempt to make sure ensureWritable copied the existing elements, I must have forgotten to port it to using empty().